src/components/publishing/image_collection_item.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react"
+import styled from "styled-components"
+
+interface FillwidthItemProps extends React.HTMLProps<HTMLDivElement> {
+  width?: number
+  height?: number
+  margin?: number
+  key: string
+}
+
+const FillwidthItem: React.SFC<FillwidthItemProps> = props => {
+  const { key, ...rest } = Object.assign({}, props)
+  return <div {...rest} key={key} />
```

Comment:
I’m not quite sure I understand what’s happening here. Isn’t this all essentially equal to the following?

```js
return <div {...props} />
```